gulpfile.js

```diff
@@ -47,10 +47,16 @@ SRC = {
 	js: []
 };
 
-/* Confing: Edit saucal.json to set your folders and domain
+/* Config: Edit projectFolders entry into package.json to set your folders and domain
 ========================================================= */
 try {
-	FOLDERS = JSON.parse( fs.readFileSync( './project-folders.json' ) );
+	var pkgJson = JSON.parse( fs.readFileSync( './package.json' ) );
+
+	if ( "projectFolders" in pkgJson ) {
```

Comment:
Need project-folders.json to take precedence.

If it doesn't exist, look for it in pkgJson.